Extensions/TileMap/JsExtension.js

```diff
@@ -656,6 +656,7 @@ const defineCollisionMask = function (
       'tilemapJsonFile',
       new gd.PropertyDescriptor(objectContent.tilemapJsonFile)
         .setType('resource')
+        .addExtraInfo('tilemap')
```

Comment:
I did not remember two types were even supported! I think you should specify only `tilemap` though, because it already allows to choose jsons?

<img width="589" alt="image" src="https://user-images.githubusercontent.com/1280130/224657850-4355b921-ec97-4f26-92af-125b49a59278.png">

Or do you need this to allow "Remove unused resources" to not remove a json resource that would be already added by existing games?

Reply:
It's needed because the property field check the resource type and old projects use json resources.
Also, in the resource manager, there is no way to know if a .json is seen as a json or a tilemap resource.

Reply:
Alright then!